src/Perspex.Controls/ContextMenu.cs

```diff
@@ -0,0 +1,162 @@
+﻿namespace Perspex.Controls
+{
+    using Input;
+    using Interactivity;
+    using LogicalTree;
+    using Primitives;
+    using System;
+    using System.Reactive.Linq;
+
+    public class ContextMenu : SelectingItemsControl
+    {
+        private bool _isOpen;
+
+        /// <summary>
+        /// Defines the ContextMenu.Menu attached property.
+        /// </summary>
+        //public static readonly AttachedProperty<object> ContextMenuProperty =
+        //    PerspexProperty.RegisterAttached<ContextMenu, Control, object>("Menu");
```

Comment:
Probably just remove this for now. I don't like comment out code in the repos ;) We can add it back later when OmniXAML is fixed.